utils/storage-usage-delta-calculator/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use near_chain_configs::Genesis;
+use near_primitives::runtime::config::RuntimeConfig;
+use near_primitives::state_record::StateRecord;
+use node_runtime::Runtime;
+use std::fs::OpenOptions;
+use std::io::prelude::*;
+use std::io::Error;
+
+fn main() -> Result<(), Error> {
+    println!("Start");
+
+    let genesis = Genesis::from_file("output.json");
```

Comment:
I feel like you expect to have a state dump in this `output.json`, but it is not clear. Please, provide more details in the doc comments as to how to use it.